spin/pom.xml

```diff
@@ -3,12 +3,13 @@
 
   <!-- camunda BPM parent release pom -->
   <parent>
-    <groupId>org.camunda.spin</groupId>
-    <artifactId>camunda-spin-bom</artifactId>
+    <groupId>org.camunda.bpm</groupId>
+    <artifactId>camunda-parent</artifactId>
+    <relativePath>../parent</relativePath>
```

Comment:
❓ What do you aim with this change? Can we still keep the bom as a parent?

Reply:
It is in order to avoid a cyclic dependency between camunda-bom and camunda-spin-bom.

spin can no longer user caminda-spin-bom as a parent, instead of the parent relationship it _imports_  it (indirectly, using camunda-bom).

Reply:
Spin bom points spin root that points now to camunda-parent. Makes sense to me now.